pkg/api/ldap_debug.go

```diff
@@ -272,22 +270,18 @@ func (server *HTTPServer) GetUserFromLDAP(c *models.ReqContext) Response {
 		IsDisabled:     user.IsDisabled,
 	}
 
-	orgRoles := []RoleDTO{}
+	orgRoles := []LDAPRoleDTO{}
 
-	for _, g := range serverConfig.Groups {
-		role := &RoleDTO{}
+	for _, ldapGroup := range user.Groups {
```

Comment:
thinking here is to only loop over returned ldap groups, as in my mind that is the most interesting thing for this table to show. What groups did ldap return and which did successfully get mapped to org roles?